examples/run/cuda/seeding_example_cuda.cpp

```diff
@@ -145,7 +145,8 @@ int seq_run(const traccc::seeding_input_config& i_cfg,
         /*time*/ auto start_tp_estimating_cuda =
             std::chrono::system_clock::now();
 
-        auto params_cuda = tp_cuda(spacepoints_cuda_buffer, seeds_cuda_buffer);
+        auto params_cuda_buffer =
```

Comment:
Instead of using `auto`, using `traccc::cuda::track_params_estimation::output_type` would sit better with me. Though this is more of a personal taste question.